### ISSUE TITLE
Button role tertiary does not show contour and background color when not on focus

### ISSUE THREAD

**Issue by IsaSih** [rancher/dashboard#15759]
**Setup**
- Rancher version: 
Rancher | v2.13-7da7e6a10e3279a6a9a4a241a636c521333d5506-head
-- | --
Dashboard | master 6ff4679
Rancher Prime CLI | v2.12.0-rc.1

[Rancher](https://github.com/rancher/rancher)	v2.13-7da7e6a10e3279a6a9a4a241a636c521333d5506-head
[Dashboard](https://github.com/rancher/dashboard)	master 6ff4679
[Rancher Prime CLI](https://github.com/rancher/cli)	v2.12.0-rc.1

- Browser type & version: Chrome

**Describe the bug**
This needs to be fixed in both suse prime and modern themes for both light and dark mode.

**To Reproduce**
Go to clusters management, select any cloud provider and observe the button role tertiary style with no contour nor background color unless hovered. This occurs on both suse and default modern themes.

**Expected Result**
Tertiary buttons should have a delimited contour and visible background without focus, as happens in the classic theme

**Screenshots**
Classic theme
<img width="1217" height="937" alt="Image" src="https://github.com/user-attachments/assets/6d2606c3-7141-4d03-a9ec-05e2b34d6d6b" />

suse prime theme

<img width="1702" height="937" alt="Image" src="https://github.com/user-attachments/assets/939f5fa1-3261-498d-8da3-6b322a498a8b" />
<img width="1702" height="937" alt="Image" src="https://github.com/user-attachments/assets/6d360b4c-443c-41ae-8a51-40c44b3e1bc4" />

modern theme

<img width="1234" height="937" alt="Image" src="https://github.com/user-attachments/assets/fcde45a3-55dd-41f3-bfe7-e868e7f003ea" />
<img width="1234" height="937" alt="Image" src="https://github.com/user-attachments/assets/13f1f85f-0c8b-4fe7-bb9f-cf541fbb8c12" />

**Comment by rak-phillip** [comment]
This should be expected for the tertiary buttons. @kwwii @oboc-sts can you confirm?

**Comment by IsaSih** [comment]
well, imo, there's a lack of consistency between the classical and modern themes then. I personally prefer the appearance of the classical theme, as for the modern theme, the texts look somewhat lost in the major component. I perceive it like a visual bug.

**Comment by kwwii** [comment]
The new theme was not developed as a new version of the old theme, but rather was intended to go in a new direction based on the brand defined for us. That being said, it is a WIP and we will take this into consideration and review the design of the teriary buttons.
